_projects/2022/100462872/100462872.Rmd

```diff
@@ -0,0 +1,21 @@
+---
+title: "Use of Social Media Platforms by demographic group in U.S 2021"
+description: |
+  A short description of the post.
+author: Marina Sánchez Moreno
+date: 2023-01-22
+output:
+  distill::distill_article:
+    self_contained: false
+---
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+Distill is a publication format for scientific and technical writing, native to the web.
+
+Learn more about using Distill at <https://rstudio.github.io/distill>.
+
+
```

Comment:
The Rmd is empty! Maybe you committed the wrong one?

Reply:
Yes! I think is ok now (I committed and pushed the correct .Rmd and .html)